<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MediaPlayer</class>
 <widget class="QWidget" name="MediaPlayer">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>877</width>
    <height>483</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidgetVideoRender" name="widRender" native="true"/>
   </item>
   <item>
    <widget class="QWidget" name="widControl" native="true">
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>44</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>18</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>19</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btnPlay">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="baseSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	border-image: url(:/Course/course/play.png);
}
QPushButtont:hover{
	border-image: url(:/Course/course/play.png);
}
QPushButton:pressed{
	border-image: url(:/Course/course/play_press.png);
}
QPushButton:disabled{
	border-image: url(:/Course/course/play.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbCurTime">
        <property name="styleSheet">
         <string notr="true">color: rgb(136, 136, 136);
font: 12pt &quot;微软雅黑&quot;;
margin-left: 5px;
margin-right: 10px;</string>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSliderCustom" name="hsProgress">
        <property name="minimumSize">
         <size>
          <width>364</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>364</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
	background: rgb(136, 136, 136);
	height: 4px;
}

QSlider::handle:horizontal {
	border-radius: 5px;
	margin: -5px -5px;
	width:  10px;	
    background: rgb(0, 110, 255);
}

QSlider::sub-page:horizontal {
	background: rgb(0, 110, 255);
}</string>
        </property>
        <property name="value">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lbTotalTime">
        <property name="minimumSize">
         <size>
          <width>80</width>
          <height>0</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">color: rgb(136, 136, 136);
font: 12pt &quot;微软雅黑&quot;;
margin-left: 10px;
margin-right: 18px;
</string>
        </property>
        <property name="text">
         <string>00:00:00</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnVolume">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	border-image: url(:/Course/course/voice.png);
}
QPushButtont:hover{
	border-image: url(:/Course/course/voice.png);
}
QPushButton:pressed{
	border-image: url(:/Course/course/volice_press.png);
}
QPushButton:disabled{
	border-image: url(:/Course/course/voice.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="minimumSize">
         <size>
          <width>10</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>10</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSliderCustom" name="hsVolume">
        <property name="minimumSize">
         <size>
          <width>66</width>
          <height>0</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>66</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QSlider::groove:horizontal {
	background: rgb(136, 136, 136);
	height: 4px;
}

QSlider::handle:horizontal {
	border-radius: 5px;
	margin: -5px -5px;
	width:  10px;	
    background: rgb(0, 110, 255);
}

QSlider::sub-page:horizontal {
	background: rgb(0, 110, 255);
}</string>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="singleStep">
         <number>5</number>
        </property>
        <property name="pageStep">
         <number>5</number>
        </property>
        <property name="value">
         <number>50</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="btnOpen">
        <property name="minimumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>22</width>
          <height>22</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QPushButton{
	border-image: url(:/Course/course/file.png);
}

QPushButton::pressed{
	border-image: url(:/Course/course/file_press.png);
}</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QWidgetVideoRender</class>
   <extends>QWidget</extends>
   <header>qwidgetvideorender.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QSliderCustom</class>
   <extends>QSlider</extends>
   <header>qslidercustom.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../res/Course.qrc"/>
 </resources>
 <connections/>
</ui>
